README.md

```diff
@@ -10,21 +10,47 @@ Copyright (C) 2013 The Nature Conservancy
 ## Static Site Conversion (**work in progress**)
 This project is actively being converted into a static site and away from using .NET on the `feature/task-1` branch. During this time, the project will continue to work off `develop` via the typical set up in Visual Studio on Windows. It may break with Visual Studio on `feature/task-1` and its child branches.
 
+*Note the static site port is 54634 instead of 54633*
+
 ### Run a python development environment
-To run the project locally, on any OS, clone the repo and run `./scripts/update` from root to set up. To run the project server thereafter, `./scripts/server`.
+To run the project locally, on any OS, clone the repo and ensure you have `Python 2.x`. You can choose to work on your host machine or within a docker container. It is advised to work within docker if you'd rather not adjust your local python setup.
+
+#### Host
+Ensure `pip` is installed.
+
+Install the python requirements:
+`python ./scripts/update.py`
```

Comment:
This may be a quirk related to my machine setup but I had to prefix this command with `sudo` to get it to work. After that it did work, however.